### PR TITLE
[General] update `[p]serverinfo 1` guild features.

### PR DESCRIPTION
**Pull request by ltzmax**
### Description of the changes
Reopening this cause i deleted the fork on previous pr (#5525)

Anyway, this PR updates the list of features in `[p]serverinfo 1` to the [updated list from discord](https://discord.com/developers/docs/resources/guild#guild-object-guild-features).

### Have the changes in this PR been tested?
Yes

### CODE REVIEW

File: redbot/cogs/general/general.py [hunk 1]
```diff
@@ -347,23 +347,27 @@ def _bitsize(num: int):
...
-                "DISCOVERABLE": _("Server Discovery"),
+                "DISCOVERABLE": _("Discoverable"),
                 "FEATURABLE": _("Featurable"),
                 "INVITE_SPLASH": _("Splash Invite"),
-                "MEMBER_LIST_DISABLED": _("Member list disabled"),
                 "MEMBER_VERIFICATION_GATE_ENABLED": _("Membership Screening enabled"),
-                "MORE_EMOJI": _("More Emojis"),
-                "NEWS": _("News Channels"),
+                "MONETIZATION_ENABLED": _("Monetization Enabled"),
+                "MORE_STICKIES": _("More Stickies"),
```
Comment (PredaaA) [inline redbot/cogs/general/general.py]: This one is `MORE_STICKERS`, so "More Stickers". (even tho stickies cute)

File: redbot/cogs/general/general.py [hunk 2]
```diff
@@ -347,23 +347,27 @@ def _bitsize(num: int):
...
+                "NEWS": _("News"),
                 "PARTNERED": _("Partnered"),
-                "PREVIEW_ENABLED": _("Preview enabled"),
-                "PUBLIC_DISABLED": _("Public disabled"),
-                "VANITY_URL": _("Vanity URL"),
+                "PREVIEW_ENABLED": _("Preview Enabled"),
```
Comment (PredaaA) [inline redbot/cogs/general/general.py:368]: It's missing `PRIVATE_THREADS` feature.
```suggestion
                "PREVIEW_ENABLED": _("Preview Enabled"),
                "PRIVATE_THREADS": _("Private Threads"),
```

**Review by PredaaA — CHANGES_REQUESTED** [review]

File: redbot/cogs/general/general.py (hunk 1 above)
Comment (ltzmax) [reply redbot/cogs/general/general.py]: Lmao thanks. i didnt even notice i typed that.

**Review by PredaaA — APPROVED** [review]
Thank you!

**Pull request merged**